templates/lacework-aws-cfg-member.template.yaml

```diff
@@ -14,13 +14,21 @@ Parameters:
       Invalid resource name prefix.  Must match pattern
       ^[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*$
   ExternalID:
-    Default: 4CEBE3B
+    Default: 4CEBE3BCEB
```

Comment:
@jefferyfry what do you think about making this a random string made of the Stack ID you mentioned? would that be a lot to add? should we just leave this as a fixed string?